### PR TITLE
Metrics update

### PR DESCRIPTION
**Pull request by jendefig**
This adds @flippedcoder 's Metric video to the end of the Get Started:  Metrics, Parameters, and Plots Doc.

### CODE REVIEW

**Comment by shcheklein** [comment]
hey, looks there are unrelated changes got into this PR?

File: content/community.json [hunk 1]
```diff
@@ -78,6 +78,22 @@
...
+    {
+      "url": "https://hasgeek.com/fifthelephant/mlops-conference/schedule",
+      "title": "HasGeek",
+      "description": "Milecia McGregor presents at HasGeek Conference from Fifth Elephant on Tuning Hyperparameters with DVC Experiments",
+      "city": "Virtual, Miami, FL",
+      "date": "2021-07-23",
+      "pictureUrl": "/img/community/events/hasgeek.png"
+    },
+    {
+      "url": "https://www.datasciencego.com/virtual",
```
Comment (iesahin) [inline content/community.json:90]: I think Jervis's talk shouldn't be here? Right?

**Review by iesahin — CHANGES_REQUESTED** [review]
I think DSGO image and Jervis's talk deserves a separate PR :) Thank you.

**Comment by jendefig** [comment]
@shcheklein @iesahin Can someone tell me how/why the contents of my previous pr got into this one? 
image

File: content/community.json (hunk 1 above)
Comment (jendefig) [reply content/community.json:90]: This was on a previous PR.  I don't know what happened, see below.

**Comment by jendefig** [comment]
Is this the proper way for me to proceed?

> If the unwanted updates are in an entirely new file that you created, then you can delete the file from your file system, then run git rm <file path> followed by git commit to remove the file from the PR.

And then then I would need to go back to main branch and git pull to restore the file?

**Comment by iesahin** [comment]
Probably you were working on previous PR's branch. You created a new branch (`metrics-update`) off of previous branch and your previous work is also attached to the changes.

Normally we base new branches to `master`, so only the changes against master is shown. So when creating a new branch:

```console
git checkout master
git pull
git checkout --branch my-new-branch
```

ensures that you're working in a new branch created with the latest `master`.

For this, if your previous PR is merged into `master`, then you should be able to

```console
git rebase -i origin/master
```

and move your branch's base to `master`. A text editor will appear that lists your commits. You can keep it intact to apply your commits to `master`. (You can do many other things in that list, it's like swiss army knife for Git commits.)


If your previous PR is not merged yet, the simplest way is to remove this extra content manually from this PR I think. (I assume your previous PR's branch was different.)

**Comment by jendefig** [comment]
> Probably you were working on previous PR's branch. You created a new branch (`metrics-update`) off of previous branch and your previous work is also attached to the changes.

Ah yes.  Skipped a step. 🤦‍♀️ And the previous was merged. Trying your advice now.  Thanks for your help @iesahin !!

**Comment by jendefig** [comment]
@iesahin Did the above. Deleted the two extra commits and closed.  Now what? I'm finding several different things online and I don't know what's right.

**Comment by iesahin** [comment]
> Deleted the two extra commits and closed. Now what?

Deleted what? I think you don't need to delete the lines in `git rebase -i`, they show the commit to be applied over `master`. It's like asking "which commits should I apply to `master`?" and you may keep your commits related to this PR in that list. :)

You're welcome :)

After that you may need to `git push --force` in your branch to update this PR.

**Comment by iesahin** [comment]
Try rebasing once more and if that doesn't work, please let me know, I can probably do that for you :)

**Comment by jendefig** [comment]
> Try rebasing once more and if that doesn't work, please let me know, I can probably do that for you :)

I deleted the two commits that were not supposed to be in metrics-update PR.  I tried above per your suggestion.  Not getting anywhere.  It's asking me to `git push --set-upstream origin metrics-update` I did that then tried the `git push --force`. And at any rate it's all a mess here.  This is beyond my capabilities at the moment @iesahin. 😞 If you can fix this it would be helpful.  All this for adding two lines of code.  I miss Squarespace! 😭 (In fairness, all this because I forgot a step, but I still miss Squarespace!)

**Comment by jendefig** [comment]
@iesahin Would it be easier to close this PR and start over?

**Comment by iesahin** [comment]
:) I think the fix `Two` to `two` you made in a file was preventing a clean rebase. @jendefig 

You can create another PR to fix typos you see. It's easier to explain that way.

It should be done now, could you check if the changes are all here. I've taken a backup of your branch and will keep it until you confirm this PR contains all :)

**Review by iesahin — APPROVED** [review]

**Review by shcheklein — APPROVED** [review]

**Comment by jendefig** [comment]
Thank you @emre and @shcheklein for your help!  All looks good!

**Pull request merged**